### DISCUSSION TITLE
Server and system notifications

### DISCUSSION THREAD

**Discussion by WolfoTheWolf** [pelican-dev/panel#42] [Ideas]
Ability for 3rd party services to be able receiving system logs for the following.

Client Area: Ability to receive server specific logs, maybe crashes, starts, stops and restarts, added users, edited users, database creation or deletion, file edits and so on. The ability for them to log this stuff via maybe a webhook, telegram, or even in email would be a nice bonus feature to have. 

Admin Area (/admin): Ability to receive notifications via a webhook or emails about the following, when wings or ptero panel reports a error, maybe a game server crashed and for what reason, instead od digging through wings logs to find it if the user reports a crash for a unknown reason, and changes made via the admin page, someone modifies a api, node or egg. 

Just would be nice to have a live log of what is going on

**Comment by lancepioch** [comment]
Yes, the idea is to implement webhooks here: https://github.com/pelican-dev/panel/issues/9
